### PR TITLE
CBG-723: Remove walrus references from build readme.txt

### PR DESCRIPTION
**Pull request by sarathkumarsivan**
- Removed basic-walrus-bucket.json, basic-walrus-persisted-bucket.json from examples.
- Modified the README.txt
- Modified Sync Gateway web console URL in build/README.Debian.tmpl
- Added num_index_replicas options in examples/basic-couchbase-bucket.json

The walrus references can be removed from both postinst.tmpl and rpm.spec.tmpl files when removing the walrus option completely.

### CODE REVIEW

File: build/README.txt [hunk 1]
```diff
@@ -1,24 +1,12 @@
 Couchbase Sync Gateway
 
-This is a self-contained installation of Couchbase Sync Gateway. You start Sync Gateway by running sync_gateway with the -url option. The argument for the -url option is the HTTP URL of the Couchbase Server to which you want Sync Gateway to connect. If you do not include any additional command-line options, the default values are used.
+This is a self-contained installation of Couchbase Sync Gateway. You can start Sync Gateway by specifying a configuration file through command-line. The configuration file determines the runtime behavior of Sync Gateway, including server configuration and the database or set of databases with which a Sync Gateway instance can interact.
```
Comment (adamcfraser) [inline build/README.txt]:
```suggestion
This is a self-contained installation of Couchbase Sync Gateway. You can start Sync Gateway by running sync_gateway and specifying a configuration file. Sample configuration files can be found in the examples folder. Modify the example configuration files to include your Couchbase Server address and credentials. 

Detailed getting started information can be found at https://docs.couchbase.com/sync-gateway/current/getting-started.html
```

File: build/README.txt [hunk 2]
```diff
@@ -1,24 +1,12 @@
...
+$ ./sync_gateway examples/basic-couchbase-bucket.json
 
 You can stop Sync Gateway with Control-C. There is no specific shutdown procedure and it is safe to stop it at any time.
 
-This application can be run from any location on any writeable volume.
-You can choose to move it to your Application directory, but this is not required. However, do not move the application while it's running.
+This application can be run from any location on any writeable volume. You can choose to move it to your Application
+directory, but this is not required. However, do not move the application while it's running.
+
+You can find more detailed documentation at https://docs.couchbase.com/sync-gateway/current/getting-started.html
```
Comment (adamcfraser) [inline build/README.txt]: (moved to the opening paragraph)

File: build/README.Debian.tmpl [hunk 3]
```diff
@@ -3,5 +3,5 @@
 
 @@PRODUCT_BASE_CAP@@ Server distribution for Debian
 
-After installing this package, please visit http://hostname:8091
+After installing this package, please visit http://localhost:4985/_admin/
```
Comment (adamcfraser) [inline build/README.Debian.tmpl]: SG's admin UI is also deprecated, shouldn't recommend it here.

**Review by adamcfraser — CHANGES_REQUESTED** [review]

File: build/README.Debian.tmpl (hunk 3 above)
Comment (sarathkumarsivan) [reply build/README.Debian.tmpl]: This change has been removed.

File: build/README.txt (hunk 2 above)
Comment (sarathkumarsivan) [reply build/README.txt]: This change has been committed.

**Review by adamcfraser — APPROVED** [review]

**Pull request merged**